admin/pg12/upgrade-to-pg12

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -e
+
```

Comment:
Even though upgrade instructions will be added to release notes or blog post, there should be at least a short description about this script. See [`admin/create-amqp-extension`](https://github.com/metabrainz/musicbrainz-docker/blob/v-2020-05-13-solr7/admin/create-amqp-extension#L5-L24) for example.

Reply:
Added in https://github.com/metabrainz/musicbrainz-docker/pull/146/commits/3a5be2ee5612919999083c2dd6a95b5bc3c9a83d

Reply:
It also fixed running the script from random path.